### PR TITLE
Adding documentation to run development in Tina

### PR DESCRIPTION
**Pull request by hirvin-faria**
Adding documentation to assist users who want to contribute to the tina project, commenting on how to run a tina development environment, so that they can make changes to the repository packages and test the changes to the demo packages.

### CODE REVIEW

File: content/docs/contributing/setting-up.md [hunk 1]
```diff
@@ -28,20 +28,32 @@ cd packages/demo-gatsby
...
+| lerna run watch | Watch all packages for rebuilds. |
 
 ## Links
 
-- [Circle CI](https://circleci.com/gh/tinacms/tinacms): Continuous Integration
+* [Circle CI](https://circleci.com/gh/tinacms/tinacms): Continuous Integration
+
+## Run Development Packages 
+
+To contribute to Tina code, after installing the development setup we can run the Demo applications contained in the _packages_ directory, how **demo-cra**, **demo-gatsby** and **demo-next**.
```
Comment (DirtyF) [inline content/docs/contributing/setting-up.md:51]:
```suggestion
After installing the development setup you can run demo applications contained in the _packages_ directory **demo-cra**, **demo-gatsby** and **demo-next**.
```

File: content/docs/contributing/setting-up.md [hunk 2]
```diff
@@ -28,20 +28,32 @@ cd packages/demo-gatsby
...
+
+These projects can be used as a development environment for Tina packages, you can edit any other package contained within Tina packages how **@tinacms** and **react-tinacms-editor**.
```
Comment (DirtyF) [inline content/docs/contributing/setting-up.md:53]:
```suggestion
These projects can be used as a development environment for Tina packages, you can edit any other package contained within Tina packages.
```

**Comment by DirtyF** [comment]
Hey thanks, there's already a [CONTRIBUTING](https://github.com/tinacms/tinacms/blob/master/CONTRIBUTING.md#development) document at the root of the repository.

**Comment by hirvin-faria** [comment]
> FYI there's also a [CONTRIBUTING](https://github.com/tinacms/tinacms/blob/master/CONTRIBUTING.md#development) document at the root of the repository.

@DirtyF Thanks for the information.

I can add the method of running Tina for Development on Troubleshooting in Development. I didn't find anything in the documentation about this

**Review by dwalkr — APPROVED** [review]

**Pull request merged**